src/panels/config/automation/trigger/types/ha-automation-trigger-webhook.ts

```diff
@@ -1,6 +1,10 @@
-import "@polymer/paper-input/paper-input";
```

Comment:
Keep this import

Reply:
[6721694](https://github.com/home-assistant/frontend/pull/11568/commits/67216944d2752b622c82f29224339800d16e9a87)